app/src/main/java/com/example/banking/AccountVM.kt

```diff
@@ -0,0 +1,62 @@
+package com.example.banking
+
+import android.app.Application
+import androidx.compose.runtime.mutableStateListOf
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.viewModelScope
+import androidx.room.Room
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+
+class AccountVM (application: Application) : AndroidViewModel(application) {
```

Comment:
Android View Model? DB in view model? change all code, use koin and provide application there

Reply:
I know a little bit about koin, but I tried to do it